packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx

```diff
@@ -3,34 +3,36 @@ import { Helmet } from 'react-helmet'
 
 import { useCurrency } from '../../../sdk/localization/useCurrency'
 import { useAsyncProduct } from '../../../sdk/product/useAsyncProduct'
+import { Props } from '../../../templates/product'
```

Comment:
maybe it is better to export this interface with another name, so it is very generic. Maybe `ProductTemplateProps`?